src/main/resources/static/js/script.js

```
@@ -0,0 +1,145 @@
+    return formatRegNumber;
+}
+
+function formatDate(date) {  // 출력, 입력 날짜 YYYY/MM/DD 형식으로 바꾸기
+    let dateString = '';
+    if(typeof date !== 'string'){  // 날짜를 입력받는 경우
+        for (let i = 0; i < date.value.length; i++) {
+            if (!isNaN(date.value[i]) && date.value[i] !== ' ') {  // 숫자만 입력받음
+                dateString += date.value[i];
+            }
+        }
+    }else{  // 조회에서 날짜를 출력하는 경우
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)